homeassistant/components/ecovacs/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "ecovacs",
-  "name": "Ecovacs",
+  "name": "Ecovacs Legacy",
```

Comment:
Yes it will still work for the old bots. Ecovacs stopped selling them years ago...

Ecovacs is using different protocols for communication:

- XMPP: oldest bots, which are supported by the `ecovacs` intgeration
-  MQTT: will be added with `ecovacs_mqtt`
	- XML: old bots (eg. 900): PR open to add support for it into `deebot_client`  (Currently supported by the custom integration [hacc-ozmo](https://github.com/Ligio/hacc-ozmo))
	- JSON: newer bots (starting with 920/950) supported by `deebot_client`. The 950 was released in 2019